packages/IMAPClient-Core.package/ICFileReadWriter.class/instance/createJsonHeaderFrom..st

```diff
@@ -8,6 +8,6 @@ createJsonHeaderFrom: aDictionary
 		'date' ->  (aDictionary at: #date ifAbsent: ['']).
 		'from' -> (aDictionary at: #from ifAbsent: ['']).
 		'subject' -> (aDictionary at: #subject ifAbsent: ['']).
-		'to' -> (aDictionary at: #to ifAbsent: ['']) }.
+		'to' -> (aDictionary at: #to ifAbsent: [''])}.
```

Comment:
In `createHeaderFrom: aJsonObject` you added a dot to the end. Here you are removing it. Should be consistent :)

Reply:
Ich hatte zunächst den Header erweitert, aber dann wieder rückgängig gemacht. Dabei habe ich wohl vergessen den Punkt zu löschen...hab's korrigiert.